### PR TITLE
Bump version and rerender

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Changes:
    MNT: Re-rendered with conda-smithy 2.4.0
    Version bump to 1.8.2

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,12 +8,12 @@ package:
...
+  sha256: 'e4181a9dc686c82cbc9bb496787d659d55582c44ad74b69924c45f80f3d513a1'  # [py27]
 
   fn: avro-python3-{{ version }}.tar.gz  # [py3k]
-  url: {{ mirror }}/apache/avro/avro-{{ version }}/py3/avro-python3-{{ version }}.tar.gz  # [py3k]
-  sha256: f25a688f1011216f73f396a5b27185d132f863cfb359be9a41878349cbb418e0  # [py3k]
+  url: {{ mirror }}/avro/avro-{{ version }}/py3/avro-python3-{{ version }}.tar.gz  # [py3k]
+  sha256: 'a3e698f09e702571bc2283471ed97f7055ae4c055a037624d04a95a320242ccf'  # [py3k]
 
 build:
   number: 0
```
Comment (isuruf) [inline recipe/meta.yaml:19]: Do you want to try [`noarch: python`](https://github.com/conda-forge/conda-forge.github.io/blob/master/src/meta.rst#building-noarch-packages)?

Comment (mariusvniekerk) [reply recipe/meta.yaml:19]: Tried it but since we use two different sources for py2 vs py3 it didn't seem to work for me locally

Comment (isuruf) [reply recipe/meta.yaml:19]: Ah, if it's 2 different sources it won't work. Sorry, I didn't see that.

Comment (mariusvniekerk) [reply recipe/meta.yaml:19]: yeah so close though :(

**Pull request merged**